manifests/patch-e2e.sh

```diff
@@ -38,20 +32,32 @@ namespace=${namespace:-tidb-admin}
 CURDIR=$(cd $(dirname ${BASH_SOURCE[0]}); pwd )
 os=`uname -s| tr '[A-Z]' '[a-z]'`
 
-CA_BUNDLE=$(kubectl get configmap -n kube-system extension-apiserver-authentication -o=jsonpath='{.data.client-ca-file}' | base64 | tr -d '\n')
-echo $CA_BUNDLE
+case ${os} in
+  linux)
+    sed -i "s/namespace: \${NAMESPACE}/namespace: ${namespace}/g" $CURDIR/webhook.yaml
```

Comment:
Will this modify the file directly? This file is checked into the git. So I think it's better to do the modification on the fly by using pipe.

Reply:
Or can we do the patch after apply to k8s?

Reply:
Currently, this would only modify the file in e2e test. I think this could be marked as an issue to find a more elegant way in a future request.

Reply:
issed in https://github.com/pingcap/tidb-operator/issues/1358